ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/server/KsqlRestApplicationTest.java

```diff
@@ -415,7 +421,7 @@ public void shouldNotInitializeUntilPreconditionsChecked() {
   }
 
   @Test
-  public void shouldConfigureRocksDBConfigSetter() {
+  public void shouldConfigureRocksDBConfigSetter() throws AbortApplicationStartException {
```

Comment:
If we make `AbortApplicationStartException` a non-checked exception we can drop all these `throws`...

Reply:
Done